src/app/collection/collection-globals.scss

```diff
@@ -1,3 +1,17 @@
 // Imports from CLARK stylesheets
 @import '~_vars.scss';
-@import "~globals.scss";
\ No newline at end of file
+@import "~globals.scss";
+
+
+$ncyte-blue: #2757A4;
+$ncyte-yellow: #FFDF21;
+$ncyte-grey: rgba(85,89,94, .6);
+
+$stats-yellow: hsl(51, 100%, 80%);
+$stats-orange: orange;
```

Comment:
There is an orange, grey, purple, and blue in the globals file.

Reply:
isnt this the globals file?

Reply:
i didnt see any, so I made some variables

Reply:
There is an orange, purple, and blue in: https://github.com/Cyber4All/clark-client/blob/master/src/_vars.scss

Reply:
I only saw light blue in the vars, i think the ncyte blue was a bit different. Was this just for the stats section?

Reply:
Yep just for stats, but I believe the stats blue was the same as NCyTE in the mocks.